src/libopensc/sm.c

```diff
@@ -158,7 +158,7 @@ sc_sm_single_transmit(struct sc_card *card, struct sc_apdu *apdu)
 
 	/* send APDU flagged as NO_SM */
 	sm_apdu->flags |= SC_APDU_FLAGS_NO_SM;
-	rv = sc_transmit_apdu(card, sm_apdu);
+	rv = card->reader->ops->transmit(card->reader, sm_apdu);
```

Comment:
This change will most likely lead to problems with other cards.

See discussion in https://github.com/OpenSC/OpenSC/issues/970